electron/gog/games.ts

```diff
@@ -392,35 +395,43 @@ class GOGGame extends Game {
       ]
     }
 
-    const { error, stderr, stdout, fullCommand } = await runGogdlCommand(
+    const fullCommand = getRunnerCallWithoutCredentials(
       commandParts,
-      {
-        env: commandEnv,
-        wrappers,
-        logMessagePrefix: `Launching ${gameInfo.title}`
-      }
+      commandEnv,
+      wrappers,
+      join(...Object.values(getGOGdlBin()))
     )
+    appendFileSync(
+      this.logFileLocation,
+      `Launch Command: ${fullCommand}
+
+Game Log:
+`
+    )
```

Comment:
Maybe this is better readable?
```suggestion
    appendFileSync(
      this.logFileLocation,
      [`LAUNCH COMMAND: ${fullCommand}`, 'GAME LOG:'].join('\n'))
```